STEP/REST.py

```diff
@@ -894,8 +908,15 @@ def list(self):
 	#________________________________________________________________________________________________
 	@args.operation(help='get classification by id')
 	@args.parameter(name='id', help='the ID of classification')
-	def get(self, id):
-		return super().get('%s/%s'%(self.base,id))
+	@args.parameter(name='inheriteddata', help='Endpoint returns inherited data when parameter is supplied with the value true')
```

Comment:
add the following parameters to the @args.parameter method so that it can be a -i flag to set

short='i', flag=True